open-sphere-base/analysis/src/main/java/io/opensphere/analysis/baseball/BaseballPanel.java

```diff
@@ -0,0 +1,377 @@
+        {
+            returnValue = BaseballUtils.formatTimeSpan((TimeSpan)value, myToolbox.getPreferencesRegistry()
+                    .getPreferences(ListToolPreferences.class).getInt(ListToolPreferences.LIST_TOOL_TIME_PRECISION_DIGITS, 0));
+        }
+        else
+        {
+            returnValue = value.toString();
+        }
+        return returnValue;
+    }
+
+    /**
+     * Clears the styles for the coordinate buttons, then sets the style for selected button.
+     *
+     * @param button the button
+     */
+    private void setButtonStyles(Button button)
+    {
+        myDDButton.setStyle("");
+        myDMSButton.setStyle("");
+        myDDMButton.setStyle("");
+        myMGRSButton.setStyle("");
+        button.setStyle(BUTTON_STYLE);
+    }
+
+    /**
+     * Sets the coordinate label based off the currently active data element
+     * and the selected coordinate format.
+     */
+    private void setCoordinates()
+    {
+        if (myActiveDataElement == null)
+        {
+            myCoordinates.setText("");
+        }
+        else if (myPositionFormat == GeographicPositionFormat.MGRS)
+        {
+            myCoordinates.setText(BaseballUtils.formatMGRS(myActiveDataElement));
+        }
+        else
+        {
+            String latitudeKey = myActiveDataElement.getDataTypeInfo().getMetaDataInfo().getLatitudeKey();
+            Object latitudeValue = myActiveDataElement.getMetaData().getValue(latitudeKey);
+        	String latitude = getValueAsString(latitudeKey, latitudeValue, myActiveDataElement);
+
+            String longitudeKey = myActiveDataElement.getDataTypeInfo().getMetaDataInfo().getLongitudeKey();
+            Object longitudeValue = myActiveDataElement.getMetaData().getValue(longitudeKey);
+            String longitude = getValueAsString(longitudeKey, longitudeValue, myActiveDataElement);
+
+            myCoordinates.setText(latitude + " " + longitude);
+        }
+    }
+
+    /**
+     * Sets the data information section based off the active data element.
+     */
+    private void setDataView()
+    {
+        if (myActiveDataElement == null)
+        {
+            myDataView.setItems(null);
+        }
+        else
+        {
+            List<Pair<String, String>> data = New.list();
+            GeographicPositionFormat tempFormat = myPositionFormat;
+            myPositionFormat = GeographicPositionFormat.DECDEG;
+            data.add(new Pair<>("Field", "Value"));
+            myActiveDataElement.getMetaData().getKeys().stream().forEach(key ->
+            {
+                Object elementValue = myActiveDataElement.getMetaData().getValue(key);
+                String pairValue = getValueAsString(key, elementValue, myActiveDataElement);
+                data.add(new Pair<>(key, pairValue));
+            });
+            myDataView.setItems(FXCollections.observableList(data));
+            myPositionFormat = tempFormat;
+        }
+        setCoordinates();
+	}
```

Comment:
indentation, don't use tabs
```suggestion
    }
```